unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh

```diff
@@ -42,8 +42,10 @@ logger_cert() {
             ;;
     esac
     finalmessage=$(echo "$now" "$mtype" "$message")
-    echo "$finalmessage" >> ${logfile}
-    echo -e "$finalmessage"
+    echo "$finalmessage" | tee -a ${logfile}
+    if [ -n "$debugEnabled" ] && [ "$1" == "-e" ]; then
+        echo -e "$finalmessage"
```

Comment:
This may print two times the same thin, you could leave only the line `echo "$finalmessage" | tee -a ${logfile}`

Reply:
Done